src/css/styles.css

```diff
@@ -297,32 +297,6 @@ iframe {
     width: 100%;
 }
 
-/* ===== Progress Bar ===== */
-.progress {
-    width: 100%;
-}
-
-.progress .progress-bar.active {
-    font-weight: 700;
-    animation: progress-bar-stripes .5s linear infinite;
-}
-
-.dotdotdot:after {
```

Comment:
Please keep `.dotdotdot` class and animation. It's still useful for text as mentioned in #107